components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -841,130 +839,55 @@ def read_and_update_settings_json(settings_file_path: pathlib.Path, updates: Dic
     )
+    server_settings_json_path = (
+        get_clp_home() / "var" / "www" / "webui" / "server" / "dist" / "server" / "settings.json"
+    )
+
+    validate_webui_config(clp_config, client_settings_json_path, server_settings_json_path)
 
-    validate_log_viewer_webui_config(clp_config, settings_json_path)
+    # Read, update, and write back client's and server's settings.json
+    client_settings_json_updates = {
+        "ClpStorageEngine": clp_config.package.storage_engine,
+        "MongoDbSearchResultsMetadataCollectionName": clp_config.webui.results_metadata_collection_name,
```

Pylint (3.3.7)

[convention] Line too long (104/100)

(C0301)